nip47/event_handler.go

```diff
@@ -93,24 +93,26 @@ func (svc *nip47Service) HandleEvent(ctx context.Context, relay nostrmodels.Rela
 		}
 	}
 
-	encryption := "nip04"
+	encryption := constants.ENCRYPTION_TYPE_NIP04
 	encryptionTag := event.Tags.GetFirst([]string{"encryption"})
+	if encryptionTag != nil {
```

Comment:
```suggestion
	if encryptionTag != nil && encryptionTag.Value() != ""{
```

Reply:
Also then have to put versioning in `else`

Reply:
Ignore the comment, if encryption is overriden to `""` because of empty value, it will result in `UNSUPPORTED_ENCRYPTION` error which is more apt.